src/index.js

```diff
@@ -8,30 +8,51 @@ import {runScripts} from './transformScriptTags';
 function processOptions(options) {
   // Parse preset names
   const presets = (options.presets || []).map(presetName => {
-    if (typeof presetName === 'string') {
-      var preset = availablePresets[presetName];
-      if (!preset) {
-        throw new Error(`Invalid preset specified in Babel options: "${presetName}"`);
+    let preset = null;
+
+    if (presetName instanceof Array && typeof presetName[0] === 'string') {
```

Comment:
See my code above - `Object.prototype.toString.call(arg) === '[object Array]'` avoids the need to polyfill `Array.isArray`, and also handles edge cases that `instanceof Array` doesn't handle :smile: